starter-code/scripts/articleView.js

```diff
@@ -74,40 +74,56 @@ articleView.setTeasers = function() {
 };
 
 articleView.initNewArticlePage = function() {
+  this.handleMainNav();
   // TODO: Ensure the main .tab-content area is revealed. We might add more tabs later or otherwise edit the tab navigation.
 
-
+  $('#article-json').hide();
   // TODO: The new articles we create will be copy/pasted into our source data file.
   // Set up this "export" functionality. We can hide it for now, and show it once we have data to export.
 
   $('#article-json').on('focus', function(){
     this.select();
   });
 
+  $("#newArticle").on("change", articleView.create);
+
   // TODO: Add an event handler to update the preview and the export field if any inputs change.
 
 };
 
 articleView.create = function() {
   // TODO: Set up a var to hold the new article we are creating.
   // Clear out the #articles element, so we can put in the updated preview
+  var newArticleData = {};
+    newArticleData.title = $("#title").val();
+    newArticleData.body = $("#body").val();
+    newArticleData.articleAuthor = $("#articleAuthor").val();
```

Comment:
should be `newArticleData.author = ...` to match the property in blogArticles.js